documentation/source/install.rst

```diff
@@ -0,0 +1,221 @@
+.. _install:
+
+************
+Installation
+************
+
+Using cocotb requires installation of prerequisites and installation of cocotb itself.
+In this document, we are assuming that you already have a
+:ref:`supported simulator<simulator-support>` available in ``PATH``.
+
+
+.. _install-prerequisites:
+
+Installation of Prerequisites
+=============================
+
+Cocotb has the following requirements:
+
+* Python 3.5+
+* Python-dev packages
+* GCC 4.8.1+ or Clang 3.3+ and associated development packages
+* GNU Make
+* A Verilog or VHDL simulator, depending on your RTL source code
+
+.. versionchanged:: 1.4 Dropped Python 2 support
+
+.. note:: In order to use a 32-bit simulator you need to use a 32-bit version of Python.
+
+The installation instructions vary depending on your operating system:
+
+.. tabs::
+
+   .. group-tab:: Windows
+
+      `Conda <https://conda.io/>`_ is an open-source package and environment management system that we recommend for Windows.
+
+      Download and install `Miniconda <https://docs.conda.io/en/latest/miniconda.html>`_ from https://conda.io/.
+      From a Command Prompt or the Windows PowerShell, use the following line to install a compiler (GCC or Clang) and GNU Make:
+
+      .. code-block::
+
+         conda install -c msys2 m2-base m2-make m2w64-toolchain libpython
+
+      .. container:: toggle
+
+          .. container:: header
+
+              **Show/Hide Example Installation Output**
+
+          The following shows an example of the output of the above installation step.
+
+          .. code-block::
+
+             <FIXME: add installation output here>
+
+   .. group-tab:: Linux - Debian
+
+      In a terminal, run
+
+      .. code-block:: bash
+
+          sudo apt-get install make gcc g++ python3 python3-dev python3-pip
+          sudo apt-get install swig  # optional, needed for one of the examples
+
+      .. container:: toggle
+
+          .. container:: header
+
+              **Show/Hide Example Installation Output**
+
+          The following shows an example of the output of the above installation step.
+
+          .. code-block::
+
+             <FIXME: add installation output here>
+
+   .. group-tab:: Linux - Red Hat
+
+      In a terminal, run
+
+      .. code-block:: bash
+
+          sudo yum install make gcc gcc-c++ libstdc++-devel python3 python3-devel python3-pip
+          sudo yum install swig  # optional, needed for one of the examples
+
+      .. container:: toggle
+
+          .. container:: header
+
+              **Show/Hide Example Installation Output**
+
+          The following shows an example of the output of the above installation step.
+
+          .. code-block::
+
+             <FIXME: add installation output here>
```

Comment:
Not sure this is all that useful

Reply:
This has been suggested by @imphil in https://github.com/cocotb/cocotb/pull/1637#issuecomment-613958024 and I like the idea - we don't need to show the full output if it's long, but in order to get a feel for what a successful step looks like, it's nice and reassuring for newbies.

Reply:
I suppose it's not harmful. In the interest of getting this merged soon though, I'd be tempted to merge without this, and add it in a follow-up.

Reply:
Ok, removed.